src/redux/categories/categories.js

```diff
@@ -0,0 +1,27 @@
+// action types
+
+export const SHOW_STATUS = 'categories/categories/SHOW_STATUS';
+
+// action creators
+
+export const showStatus = () => (dispatch) => {
+  dispatch({
+    type: SHOW_STATUS,
+    payload: ['under construction'],
+  });
+};
+
+// reducer
+
+const initialState = [];
+
```

Comment:
- [ ] In this case, your initialState :arrow_down: should be an empty string as shown below because state will deal with an string :thinking:
`const initialState = '';`

Reply:
resolved